changelog/binaryReverseArgs.dd

```diff
@@ -0,0 +1,4 @@
+Deprecated `std.functional.binaryReverseArgs was removed
```

Comment:
Missing backquote

Reply:
Whoops => #7291 

Is it possible to generate a preview of the entire changelog?

Reply:
It is also automatically generated on DAutoTest, e.g. http://dtest.dlang.io/artifact/website-08a5fe1c6137cc5879028d65e8b1ee42781468bc-a53a4a2d720fb1bbe0aa8dcd509e9797/web/changelog/pending.html for this PR